pegasus/sites.v3/code.org/views/lms_skinny_banner.haml

```diff
@@ -0,0 +1,14 @@
+- if !!DCDO.get('show-updated-lms-content', false)
+  %link{href: "/css/generated/lms-skinny-banner.css", rel: "stylesheet"}
+
+  %aside.lms-banner
+    .wrapper.flex-container.justify-space-between.align-items-center.wrap
+      .text-wrapper
+        %h5.white
+          =hoc_s('lms_page.skinny_banner.heading')
+        %p.white.no-margin-bottom
+          =hoc_s('lms_page.skinny_banner.desc', markdown: :inline, locals: {canvas: 'Canvas', schoology: 'Schoology'})
+      .flex-container.justify-center.align-items-center.wrap-reverse{style: "column-gap: 5rem;"}
+        %img{src:"/images/marketing/lms-integration-icons.png", alt: ""}
+        %a.link-button.white{href: "/lms", target: "_blank", rel: "noopener noreferrer", "aria-label": hoc_s('admins_page_lms_call_to_action')}
+          =hoc_s('call_to_action_learn_more')
```

Comment:
The Music Lab skinny banner is a better one to copy from for this banner instead of the Poetry banner. We had to account for the bleeding image on the Poetry banner, but since this banner is all floaty it can be simpler 😄. 

You can see the `music_lab_skinny_banner` view [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/music/music_lab_skinny_banner.haml) and the css [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/css/music-lab-skinny-banner.scss) (ignore the `.afe` stuff). 

I made a note to add this type of banner to the Pegasus Docs for the future!